blog/content/post/2019-06-21-rte-new-version-and-kit.md

```diff
@@ -0,0 +1,128 @@
+| RS232 + COM                 | RS232 DB9 port (J14)         |
+| EXT + COM                   | 3.3V UART header (J18)       |
+
+![uart-header](/img/rte-uart-header.jpg)
+
+* eliminated issue with J6 USB port (unreliable detection of USB devices)
+* smaller capacitors were applied near MAX3232 SOIC
+* added LED indicator (D5) for relay state information
+* added easy accessible reset button (SW1) for resetting the RTE board
+
+![reset-button](/img/rte-reset-btn.jpg)
+
+* added `Open hardware` icon on PCB board
+
+![open-hardware-pcb](/covers/rte-new-revision.jpg)
+
+## Upgraded full kit
+
+We came to the conclusion to remove shop kit options for RTE. From now on there
+is only one possible set to buy - the full one with all required elements for
+development and testing devices. Moreover, we have expanded kit with M3 plastic
+spacers, set of jumpers (serial communication header) and Pomona 8-pin SOIC clip
+with 8 connection wires for flashing firmware on platforms without SPI header.
+
+To sum up, full kit costs now 93€ and includes:
+
+| Category     | Description                                                     | Quantity |
+|:------------:|:---------------------------------------------------------------:|:--------:|
+| shield       | Remote Testing Environment v1.1.0                               | 1        |
+| control unit | Orange Pi Zero 256MB RAM version                                | 1        |
+| power supply | MicroUSB 5V/2A                                                  | 1        |
+| storage      | SanDisk 16GB microSD card (with preinstalled compatible system) | 1        |
+| clip         | Pomona 8-pin SOIC clip for Device Under Test SPI interface      | 1        |
+| cables       | standard female-female connection wire 2.56mm raster            | 8        |
+| cables       | IDC 8-pin wires for Device Under Test SPI interface             | 1        |
+| cables       | DC Jack - DC Jack power cable for Device Under Test             | 1        |
+| cables       | RS232 D-Sub 9P/9P cable for serial communication                | 1        |
+| jumpers      | jumper for UART OUTPUT SELECT header                            | 2        |
+| spacers      | Polyamide M3 spacers and bolts                                  | 4        |
+
+## Summary
+
+Don't hesitate and check out our **RTE** product on the [3mdeb-shop](https://shop.3mdeb.com/product/rte/)
+where you can find more information about Orange Pi Zero shield or read about
+products related to platform security such as **Trusted Platform** modules.
+
+If you think that RTE is the product that You are looking for, but missing
+something crucial for your project or that we can help in improving the security
+of your firmware or you looking for someone who can boot your product by
+leveraging advanced features of used hardware platform, feel free to
+[boot a call with us](https://calendly.com/3mdeb/consulting-remote-meeting)
```

Comment:
```suggestion
[book a call with us](https://calendly.com/3mdeb/consulting-remote-meeting)
```
@artur-rs this should be changed in template

Reply:
It is already fixed, I have written this post on the previous template